### DISCUSSION TITLE
Can we set via API custom user_created, user_updated fields?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3015] [Archive]
Hi.

I was wondering if a user with more roles can create an item via API and set `user_created` and `user_updated` of another user?

**Comment by botzill** [comment]
OK, I did solve this by changing the user_created to `Do Nothing` on create. This way we can set custom user.

**Reply by jmechevarria** [reply]
Hi, is this option still available in the Directus web interface? Thanks
